### PR TITLE
Avoid using make install in build

### PR DESCRIPTION
**Pull request by chalasr**

### CODE REVIEW

File: src/Handler/Build.php [hunk 1]
```diff
@@ -44,7 +44,7 @@ public function __construct($cwd)
         $vagrantTasks = [
             'up --no-provision',
             'provision',
-            'ssh -- "cd /srv/app && make install"',
+            'ssh -- "cd /srv/app && composer install --verbose && bin/console doctrine:database:create --if-not-exists"',
```
Comment (ogizanagi) [inline src/Handler/Build.php:47]: Shouldn't this depend on the env that was setup ?

Comment (chalasr) [reply src/Handler/Build.php:47]: You're right, maybe calling the makefile stay the simpler way... as it is created per env.
If you have some suggestion... I take :)

Comment (ogizanagi) [reply src/Handler/Build.php:47]: Looks like something that could be configured per env in a [`manala.yml` file](https://github.com/manala/manala/pull/29#issuecomment-250901383) under a `build.install` key ? 😄

Comment (chalasr) [reply src/Handler/Build.php:47]: Really nice input!

**Comment by chalasr** [comment]
Fixed in #29

**Pull request closed without merging**